experimental/auth-logic/wrappers/rekor_wrapper.go

```diff
@@ -162,6 +164,41 @@ func checkInclusionProof(logEntryAnon *models.LogEntryAnon, registry strfmt.Regi
 	return logEntryAnon.Validate(strfmt.Default)
 }
 
+func verifySignedEntryTimestamp(logEntryAnon *models.LogEntryAnon, rekorPublicKeyBytes []byte) error {
+	// Get ECDSA key from rekord public key bytes
+	rekorEcdsaKey, err := pubKeyBytesToECDSA(rekorPublicKeyBytes)
+	if err != nil {
+		return fmt.Errorf("could not parse ecdsa key rekord public key bytes")
+	}
+
+	// Get hash of signature data
+	signatureData := models.LogEntryAnon{
+		IntegratedTime: logEntryAnon.IntegratedTime,
+		LogIndex:       logEntryAnon.LogIndex,
+		Body:           logEntryAnon.Body,
+		LogID:          logEntryAnon.LogID,
+	}
+	marshaledSignatureData, err := signatureData.MarshalBinary()
+	if err != nil {
+		return fmt.Errorf("could not marshal signature data for SignedEntryTimestamp: %v. %v", signatureData, err)
+	}
+	canonicalized, err := jsoncanonicalizer.Transform(marshaledSignatureData)
+	if err != nil {
+		return fmt.Errorf("could not canonicalize signature data for SignedEntryTimesStamp: %v. %v", signatureData, err)
+	}
+	signatureDataHash := sha256.Sum256(canonicalized)
```

Comment:
Is it safe to assume that `sha256` is always the correct hashing algorithm? Is it documented somewhere?

Reply:
Is this OK?

Reply:
The comment makes sense, but it is added to `compareEndorsementAndRekorHash`. This one is a different field. Does the same comment apply here? Perhaps add a single line comment here too.

Reply:
OK I was looking at the wrong place. I added this to the right place:
```

	// Get signature
	// There does not appear to be documentation in the rekor code that
	// specifies the hash format in models.LogEntryAnon.Verification
	// or any fields that allow the hashing algorithm to be configured.
	// From testng, it seems that Sha256 is supported.`
```